lang/en/quizaccess_seb.php

```diff
@@ -90,5 +90,7 @@
 // Capabilities.
 $string['seb:bypassseb'] = 'Bypass the requirement to view quiz in Safe Exam Browser.';
 
-// Quiz access errors.
+// Quiz access information.
 $string['invalidkeys'] = "The config key or browser exam keys could not be validated. Please ensure you are using the Safe Exam Browser with correct configuration file.";
+$string['sebrequired'] = "This quiz has been configured so that students may only attempt it using the Safe Exam Browser.";
+$string['downloadbutton'] = '<a class="btn btn-secondary mx-1" role="button" href="{$a->link}">{$a->text}</a>';
```

Comment:
Not sure if this is a good idea to keep HTML code here. Let's move it to template? and move strings like 'text' => 'Launch Safe Exam Browser' to lang strings